internal/scanner/scanner.go

```diff
@@ -110,17 +109,19 @@ func auditResource(resource *resources.AuditableResources, resourcesFetcher *Res
 		}
 		for _, resource := range resource.Resources {
 			admissionRequest := resources.GenerateAdmissionReview(resource)
-			auditResponse, err := sendAdmissionReviewToPolicyServer(url, admissionRequest, httpClient)
-			if err != nil {
-				// TODO what's the better thing to do here?
-				log.Error().Err(err)
+			auditResponse, responseErr := sendAdmissionReviewToPolicyServer(url, admissionRequest, httpClient)
+			if responseErr != nil {
+				// log error, will end in PolicyReportResult too
+				log.Error().Err(responseErr)
 				continue
 			}
 
 			log.Debug().Dict("response", zerolog.Dict().
 				Str("uid", string(auditResponse.Response.UID)).
 				Bool("allowed", auditResponse.Response.Allowed)).
 				Msg("audit review response")
```

Comment:
wrap that into an `else { }` block, otherwise things will go south when `responseErr != nil`

Reply:
Indeed. Done in https://github.com/kubewarden/audit-scanner/pull/40/commits/2db512b545e1262c1288a96bc55252ee2ac676ca.